normandy/control/static/control/js/components/action_forms/HeartbeatForm.jsx

```diff
@@ -9,14 +10,17 @@ class SurveyForm extends React.Component {
     const { selectedSurvey, fields, showDefaults } = this.props;
     const surveyObject = selectedSurvey || fields.defaults;
     let headerText = 'Default Survey Values';
-    let containerClass = 'fluid-8';
+    let containerClasses = classNames({
+      'fluid-8': true,
```

Comment:
\<nit\>You can just pass `'fluid-8'` as a positional argument and it'll work. \</nit\>